src/pages/api/breadcrumbs.ts

```diff
@@ -74,8 +78,12 @@ async function fetchElements(
   fieldName: string,
   fieldValue: string,
   orgId: string,
+  req: NextApiRequest,
   apiFetch: ApiFetch
 ): Promise<BreadcrumbElement[]> {
+  const lang = getBrowserLanguage(req);
+  const messages = await getServerMessages(lang, messageIds);
```

Comment:
Using these should not be necessary (and the `req` not required here). Instead of `label`, use `labelMsg` when you want to use a translated string.

Reply:
Good to know, but with the utility function now I don't even need to translate the string at this level, so that's better!